Library/Formula/pcre.rb

```diff
@@ -24,6 +24,7 @@ class Pcre < Formula
   def install
     ENV.universal_binary if build.universal?
 
+    system "./autogen.sh" if build.head?
```

Comment:
Presumably this needs the autotools, in which case you'll need to add a `head` block with the autotools as build dependencies.